### PR TITLE
Hello (and a cat pic) from new student

### PR DESCRIPTION
**Pull request by ses3332002**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1456,3 +1456,6 @@ Thank you for the new course!
 ## Hi Kottans!
 My name is [Evgenii](https://github.com/evgenii-del). I want to be a Full Stack web developer. \
 ![cat-image](assets/images/cute_cat_kottans.jpg)
+
+## Hi! I'm Sergey, and here is Shakespeare, he's very british :)
+![cat-image](assets/images/shakespeare.jpg)
```
Comment (nazmariam) [inline README.md:1461]: Hi Sergey and Shakespeare! Good job!

**Review by nazmariam — APPROVED** [review]

**Review by Temu4 — APPROVED** [review]
Looks good to me! 👍

**Review by Temu4 — CHANGES_REQUESTED** [review]
Hey, well done! 😃 
Unfortunately you haven't only added your greeting but made changes in many places in the readme file.
I guess you have some auto-formatting settings like **Format On Save** in you redactor/IDE. And when you save file it automatic formats. 
Try to disable it, restore readme for initial version and add your greetings again.

Please, fill free to ping me if you need more info 🐱

File: README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ Hi,
 
 This is a training repository for Kottans [frontend](https://github.com/Kottans/frontend),
 [backend](https://github.com/Kottans/backend),
-and [git](https://github.com/Kottans/git-course) courses. 
+and [git](https://github.com/Kottans/git-course) courses.
```
[the remark this answers is not in the document]
Comment (ses3332002) [reply README.md:9]: Please check my new attempt

File: README.md [hunk 3]
```diff
@@ -1740,4 +1740,7 @@ I'm [Alyona](https://github.com/askryp).
 ---
 # Hello everybody!
 I'm [Serge](https://github.com/sergesd). It's really nice to get new and important knowledge in such a nice format and see a lot of progress from everybody on the Telegram channel. It's inspiring!
-![Cat](assets/images/CatFromLviv.jpg)
\ No newline at end of file
+![Cat](assets/images/CatFromLviv.jpg)
```
Comment (ses3332002) [inline README.md:1743]: Is there another whitespace deleted?
I turned off this option in my editor, so it shouldn't have done it.

**Review by miroslav-kolomiets — APPROVED** [review]
Nice work! But for the future it is better to avoid force-push ;)

**Comment by ses3332002** [comment]
Thank you so much!
I've learned that I need to put empty lines at the end of files, and that
my editor should not edit other people's lines, removing final spaces from
there.
But what was my last mistake? Why GitHub shows that some foreign line was
deleted and then added again?

пн, 2 нояб. 2020 г. в 23:04, Miroslav Kolomiets <notifications@github.com>:

**Pull request merged**